Source/Laser/LaserProfiles.H

```diff
@@ -0,0 +1,156 @@
+#ifndef WARPX_LaserProfiles_H_
+#define WARPX_LaserProfiles_H_
+
+#include <AMReX_REAL.H>
+#include <WarpXParser.H>
+#include <AMReX_ParmParse.H>
+#include <AMReX_Vector.H>
+
+#include <WarpXParser.H>
+
+#include <map>
+#include <string>
+#include <memory>
+#include <functional>
+#include <limits>
+
+namespace WarpXLaserProfiles {
+
+//_________________________________________Laser parameters shared among classes
+struct CommonLaserParameters
+{
+    amrex::Real wavelength;
+    amrex::Real e_max;
+    amrex::Real theta_stc;
```

Comment:
```suggestion
    amrex::Real theta_stc;  //! ... ?? this is a greek variable :)
```

Reply:
I've added a comment. Actually, `theta_stc` is used for now only by the `Gaussian` laser profile. However, I've put it here since it is calculated using quantities (`P_X` and `stc_direction`) which are not passed to the LaserProfile classes...